user-research/contributor-research/Introduction.md

```diff
@@ -0,0 +1,31 @@
+## Abstract:
+Knative is a vital tool in the open-source and cloud-native ecosystem, enabling the development of enterprise-level serverless and event-driven applications. However, its complexity poses challenges for both new contributors and maintainers. Providing guidance consumes valuable maintainer time, resulting in challenges when contributors leave after limited engagement.
+This research aims to understand why some individuals sustain active involvement in Knative while others contribute sporadically or disengage entirely. The ultimate goal is to identify aspects of the contributor experience that could be improved to enhance contributor retention and propose actionable recommendations to tackle these issues.
+
+## Research Goals:
+- Understand the current contributor journey within Knative.
+- Identify factors that contribute to contributors staying or leaving the Knative community.
+- Highlight areas of the contributor experience that can be improved to increase contributor retention and propose ways to improve those areas.
+
+## Objectives:
+- Gather insights into the onboarding process for new contributors to Knative.
+- Explore the experiences of both retained and departed contributors.
+- Identify pain points and areas of dissatisfaction throughout the contributor journey.
+- Investigate the effectiveness of existing resources and documentation for contributors.
```

Comment:
Just a thought: Explore with Existing Literature

Briefly mentioning existing research or literature on open-source contributor engagement and retention could be helpful. Maybe there are some good research or good practice happening in other open source projects.

Reply:
Just added a whole new 'Methods' section from my drafts!